### PR TITLE
Assignment 1 - Codes written for script #1-#8 & tested for expected execution

### PR DESCRIPTION
**Pull request by Vleung3782**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Making directories, Moving directories & files, Copying directories & files, Removing files, Creating a text file to display the list of files from another directory etc. Using wildcards is another maneuver to include big number of files. 

## What did you learn from the changes you have made?
Submission is done using VS Code. It seems to be much more friendly and intuitive than using the Git terminal prompt i.e. Git Add, Git Status, Git commit, Git checkout, Git push etc.

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
As a newbie without prior coding experience and not totally computer literate, I can't think of any. 

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
It is not extremely hard to understand the concepts behind the workflow or maneuver, it is more about getting use to the codes, retain them and the little things that a non-programmer doesn't pay attention to. Sometimes, I forgot which directory I am in resulting in 'no such file or directory'. Try to be more careful and pay attention to what the current directory is or pwd it to check. Another challenge is that I am a slow typist. It is hard to follow along in class.

## How were these changes tested?
Using "ls" a lot to list the files & directory is extremely helpful. 
Using "cat" to preview the content of the text file is useful.

## Checklist
- [X] I can confirm that my changes are working as intended

### CODE REVIEW

**Comment by pedram-aliniaye-asli** [comment]
You did a good job here!
A few things to notice:
1) Moving to the correct directory: You are changing directories (cd) and using paths like `desktop/shell/02_activities/assignments`, but it's important to ensure you're in the right directory relative to the root of your project. This part might create errors if the path isn't set up as expected.

2) Listing files and appending to inventory.txt: The command` ls ./data/processed/* >> ./data/inventory.txt` won't list files from subdirectories, as the wildcard * only matches files directly in the processed directory, not in its subdirectories.

3) Please work on the second part of the assignment which is about resolving the conflicts.
Please go ahead and do the modifications and then let me know. Thanks!

**Comment by Vleung3782** [comment]
Hi Pedram, 

I am so grateful you reached out to me, as I am stuck. Wonder if there is a good time to connect. Many thanks!!

Victor

**Comment by pedram-aliniaye-asli** [comment]
You did a great job here!
Just the last step is to work on the second part of the assignment which is about resolving the conflicts.
Please go ahead and do the modifications and then let me know. Thanks!

**Comment by monzchan** [comment]
@Vleung3782 Well done and perfect!

**Review by monzchan — APPROVED** [review]
Your effort is highly appreciated! great job!
